scripts/fast-reboot

```diff
@@ -483,10 +512,18 @@ systemctl stop swss
 if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
     initialize_pre_shutdown
 
+    if [[ "x$sonic_asic_type" == x"mellanox" ]]; then
+        check_issu_bank_file
```

Comment:
>check_issu_bank_file [](start = 8, length = 20)

This check is before `request_pre_shutdown`. If you find anything wrong, you may still `exit` the script and control/data plane are still running.

Reply:
Fixed